Fw/Logger/test/ut/LoggerRules.cpp

```diff
@@ -65,51 +65,51 @@ void LogGood::action(MockLogging::FakeLogger& truth) {
             correct = "No args";
             break;
         case 1:
-            Fw::Logger::log("One arg: %llu", ra[0]);
-            correct.format("One arg: %llu", ra[0]);
+            Fw::Logger::log("One arg: " PRI_U64, ra[0]);
```

Comment:
You need the percent sign before the closing quotation mark: `"One arg: %" PRI_U64`. The`PRI_U64` gets expanded to `"lu"` or "llu"` as appropriate for the platform.